lib/croutons/breadcrumb_trail.rb

```diff
@@ -15,16 +15,16 @@ def initialize(template_identifer, objects = {})
       build_breadcrumbs
     end
 
-    def method_missing(name, *args)
+    def method_missing(name, *args, &block)
```

Comment:
If we're going to explicitly take a block here, we should probably pass it along on line 20.